lib/bolts.js

```diff
@@ -90,11 +94,39 @@ const getSocialPoints = async (address) => {
         })
 
         return {
-            galxe: user?.galxePoints > 0 ? true : false,
-            zealy: !user?.zealyPoints > 0 ? true : false
+            galxe: user?.galxePoints > 0,
+            zealy: !user?.zealyPoints > 0
         }
     } catch (error) {
         console.log(error)
         return { galxe: null, zealy: null }
     }
+}
+
+const getUserPointsByCampaign = async (address) => {
```

Comment:
This should be in the fuul file, not here.

Reply:
I ended up deleting this since it was an unnecessary call and not used elsewhere